### PR TITLE
Revert the New "Learn By Examples" Layout

### LINKED ISSUE #2952: Revert the New LBE View Post SLBeta3
**Description:**
Revert the new LBE view post SLBeta3.

**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Revert the new "Learn By Examples" layout.
> Fixes #2952 

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: _data/all-bbes.json [hunk 1]
```diff
@@ -1353,22 +1219,6 @@
                 "disablePlayground": true,
                 "isLearnByExample": false
             },
-            {
```
Comment (MaryamZi) [inline _data/all-bbes.json:1356]: These stdlib changes aren't relevant?

Comment (praneesha) [reply _data/all-bbes.json:1356]: Added them back.

File: _data/all-bbes.json [hunk 2]
```diff
@@ -1354,6 +1220,7 @@
                 "isLearnByExample": false
             },
             {
+
```
Comment (MaryamZi) [inline _data/all-bbes.json:1223]:
```suggestion
```

File: _data/all-bbes.json [hunk 3]
```diff
@@ -1571,6 +1438,7 @@
                 "verifyOutput": false,
                 "isLearnByExample": false
             },
+
```
Comment (MaryamZi) [inline _data/all-bbes.json:1441]:
```suggestion
```

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**